.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+
+💐
+
+/cc @zendesk/dingo
+
+### Description
+
+### References
+* JIRA: https://zendesk.atlassian.net/browse/
+
+#### Before merging this PR
+- [ ] Fill out the Risks section
+- [ ] Get two approvals from the Apps teams
```

Comment:
I don't think Apps teams makes sense for an open source repo either. Besides, the rules will enforce two approvals one of which must be from a codeowner so perhaps that is enough?